### ISSUE TITLE
error client is closed 

### ISSUE THREAD

**Issue by jelmer05** [SamKirkland/FTP-Deploy-Action#354]
on: push
name: 🚀 Deploy website on push
jobs:
  web-deploy:
    name: 🎉 Deploy
    runs-on: ubuntu-latest
    steps:
    - name: 🚚 Get latest code
      uses: actions/checkout@v2
    
    - name: 📂 Sync files
      uses: SamKirkland/FTP-Deploy-Action@4.3.3
      with:
        server: n9w7ju5.lb.shared.prod.hostnet.nl
        username: p570002
        password: ${{ secrets.ftp_password }}
        timeout: 60000

Run SamKirkland/FTP-Deploy-Action@4.3.3
----------------------------------------------------------------
🚀 Thanks for using ftp-deploy. Let's deploy some stuff!   
----------------------------------------------------------------
If you found this project helpful, please support it
by giving it a ⭐ on Github --> https://github.com/SamKirkland/FTP-Deploy-Action
or add a badge 🏷️ to your projects readme --> https://github.com/SamKirkland/FTP-Deploy-Action#badge
----------------------------------------------------------------
No file exists on the server "./.ftp-deploy-sync-state.json" - this must be your first publish! 🎉
The first publish will take a while... but once the initial sync is done only differences are published!
If you get this message and its NOT your first publish, something is wrong.
----------------------------------------------------------------
Local Files:	25
Server Files:	0
----------------------------------------------------------------
Calculating differences between client & server
----------------------------------------------------------------
📁 Create: assets
📁 Create: assets/css
📁 Create: assets/images
📁 Create: assets/js
📁 Create: cv
📄 Upload: .DS_Store
📄 Upload: assets/.DS_Store
📄 Upload: assets/css/.DS_Store
📄 Upload: assets/css/main.css
📄 Upload: assets/images/.DS_Store
📄 Upload: assets/images/jelmer.jpg
📄 Upload: assets/images/jelmer.webp
📄 Upload: assets/images/work-1.jpg
📄 Upload: assets/images/work-2.jpg
📄 Upload: assets/images/work-3.jpg
📄 Upload: assets/images/work-details-image-1.jpg
📄 Upload: assets/images/work-details-image-2.jpg
📄 Upload: assets/js/database.js
📄 Upload: assets/js/main.js
📄 Upload: CNAME
📄 Upload: cv/CV jelmer van der Veen  kopie.pdf
📄 Upload: index.html
📄 Upload: README.md
📄 Upload: test.html
📄 Upload: works-setails.html
----------------------------------------------------------------
Making changes to 25 files/folders to sync server state
Uploading: 3.75 MB -- Deleting: 0 B -- Replacing: 0 B
----------------------------------------------------------------
creating folder "assets/"

----------------------------------------------------------------
--------------  🔥🔥🔥 an error occurred  🔥🔥🔥  --------------
----------------------------------------------------------------

----------------------------------------------------------------
----------------------  full error below  ----------------------
----------------------------------------------------------------

Error: Client is closed
    at /home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:5199:29
    at new Promise (<anonymous>)
    at FTPContext.handle (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:5179:16)
    at Client.sendIgnoringError (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:4227:25)
    at Client._openDir (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:4745:20)
    at Client.ensureDir (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:4736:24)
    at /home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:3555:126
    at Generator.next (<anonymous>)
    at /home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:3540:71
    at new Promise (<anonymous>)
Closing reason: Error: Timeout (control socket)
    at Socket.<anonymous> (/home/runner/work/_actions/SamKirkland/FTP-Deploy-Action/4.3.3/dist/index.js:5288:58)
    at Object.onceWrapper (node:events:509:28)
    at Socket.emit (node:events:390:28)
    at Socket._onTimeout (node:net:501:8)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7) {
  code: '0'
}
Error: Error: Client is closed

**Comment by SamKirkland** [comment]
Closing as duplicate of #106
